### PR TITLE
- [libgettext] fix RelWithDebInfo build

### PR DESCRIPTION
**Pull request by SSE4**
see https://github.com/conan-io/conan/issues/9594

Specify library name and version:  **libgettext/all**

This is also a good place to share with all of us **why you are submitting this PR** (specially if it is a new addition to ConanCenter): is it a dependency of other libraries you want to package? Are you the author of the library? Thanks!

---

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/libgettext/all/conanfile.py [hunk 1]
```diff
@@ -92,7 +96,7 @@ def build(self):
             elif self.settings.arch == "x86_64":
                 host = "x86_64-w64-mingw32"
                 rc = "windres --target=pe-x86-64"
-            automake_perldir = tools.unix_path(os.path.join(self.deps_cpp_info['automake'].rootpath, "bin", "share", "automake-1.16"))
+            automake_perldir = tools.unix_path(os.path.join(self.deps_cpp_info['automake'].rootpath, "res", "automake-1.16"))
```
Comment (madebr) [inline recipes/libgettext/all/conanfile.py:99]: Use `AUTOMAKE_LIBDIR`.
https://github.com/conan-io/conan-center-index/blob/451c44fb86c4a317ef1c79d68909f77f175ac4cb/recipes/automake/all/conanfile.py#L53-L59
https://github.com/conan-io/conan-center-index/blob/451c44fb86c4a317ef1c79d68909f77f175ac4cb/recipes/automake/all/conanfile.py#L121-L123

**Comment by madebr** [comment]
This recipe is deprecated.
Why are we updating this recipe?

**Comment by SSE4** [comment]
> This recipe is deprecated.
> Why are we updating this recipe?

it seems to be broken, see the issue above

**Comment by madebr** [comment]
I would make this a good opportunity to remove this recipe.
If it's broken + deprecated, you should really update your configuration.
People can't just assume that others will do the hard work to keep their setup working for them.

**Comment by SSE4** [comment]
> I would make this a good opportunity to remove this recipe.
> If it's broken + deprecated, you should really update your configuration.
> People can't just assume that others will do the hard work to keep their setup working for them.

fair enough :)

**Comment by madebr** [comment]
Perhaps this needs a bigger discussion with more people involved.

**Comment by Croydon** [comment]
> ould make this a good opportunity to remove this recipe.
> If it's broken + deprecated, you should really update your configuration.
> People can't just assume that others will do the hard work to keep their setup working for them.

I'm already not a fan of removing version information from git, I would dislike it even more if we are starting to remove recipes. I would like to keep it as easy as possible to reproduce the CCI remote from the current git checkout

**Comment by SSE4** [comment]
we're not removing recipes. we can stop building them at some point, but they will be there (in the repository).

**Comment by madebr** [comment]
How about this interpretation for deprecated ->no support from the "staff". Fixes are welcome though.
And indeed, let's keep the recipe in the repo.

re gettext, indeed. gettext contains only a binary, whereas libgettext packages a library.

**Pull request closed without merging**